sklearn/manifold/_t_sne.py

```diff
@@ -597,6 +598,19 @@ class TSNE(BaseEstimator):
 
         .. versionadded:: 0.22
 
+    square_distance : {True, 'legacy'}, default='legacy'
+        Whether distance values are squared before t-SNE is run. ``'legacy'``
```

Comment:
It's unclear from this formulation whether we mean "the distances will be squared internally" or "the distances are expected to be squared"

I'd suggest "Whether to square distance values before..."

Reply:
Ah, very good point. I appreciate the suggestion. :)